### PR TITLE
Update channels.csv

### PR DESCRIPTION
**Pull request by boeder8**
Added several updated logos for misc. Arab channels. Some logos courtesy of RevGear.

Logos updated for:

ARTAflam1.sa
ARTAflam2.sa
ARTCinema.sa
ARTHekayat.sa
ARTHekayat2.sa
AbuDhabiTV.ae
AlRayyanTV.qa
EmaratTV.ae
DubaiTV.ae
PNCFilm.eg
PNCFood.eg

### CODE REVIEW

File: data/channels.csv [hunk 1]
```diff
@@ -1331,19 +1331,19 @@ Arroyo.us,Arroyo,,,,US,US-CA,Pasadena,s/US-CA,eng,general,FALSE,,,,https://www.p
 ArroyoGrandeChannel20.us,Arroyo Grande Channel 20,,,,US,US-CA,Arroyo Grande,s/US-CA,eng,,FALSE,,,,https://slo-span.org/channels/arroyo-grande-channel-20,https://i.imgur.com/ZTaEFWY.png
 Arryadia.ma,Arryadia,,,SNRT,MA,,,c/MA,ara,sports,FALSE,2006-09-16,,,https://arryadia.snrt.ma/,https://i.imgur.com/Bckv3Qu.png
 Arsenal.ru,Arsenal,Арсенал,,,RU,,,c/RU,rus,,FALSE,,,,https://arsenal-tv.tv/,https://i.imgur.com/DBpVeb0.png
-ARTAflam1.sa,ART Aflam 1,,,,SA,,,c/SA,ara,,FALSE,,,,https://www.artonline.tv/,https://i.imgur.com/OKzWo0r.png
-ARTAflam2.sa,ART Aflam 2,,,,SA,,,c/SA,ara,,FALSE,,,,https://www.artonline.tv/,https://i.imgur.com/DqzhSi8.png
+ARTAflam1.sa,ART Aflam 1,,,,SA,,,c/SA,ara,,FALSE,,,,https://www.artonline.tv/,https://i.imgur.com/2lKqOjj.png
+ARTAflam2.sa,ART Aflam 2,,,,SA,,,c/SA,ara,,FALSE,,,,https://www.artonline.tv/,https://i.imgur.com/st7VS0B.png
```
Comment (freearhey) [inline data/channels.csv:1335]: What's wrong with the old logos?

File: data/channels.csv [hunk 2]
```diff
@@ -1331,19 +1331,19 @@ Arroyo.us,Arroyo,,,,US,US-CA,Pasadena,s/US-CA,eng,general,FALSE,,,,https://www.p
...
 Arryadia.ma,Arryadia,,,SNRT,MA,,,c/MA,ara,sports,FALSE,2006-09-16,,,https://arryadia.snrt.ma/,https://i.imgur.com/Bckv3Qu.png
 Arsenal.ru,Arsenal,Арсенал,,,RU,,,c/RU,rus,,FALSE,,,,https://arsenal-tv.tv/,https://i.imgur.com/DBpVeb0.png
-ARTAflam1.sa,ART Aflam 1,,,,SA,,,c/SA,ara,,FALSE,,,,https://www.artonline.tv/,https://i.imgur.com/OKzWo0r.png
-ARTAflam2.sa,ART Aflam 2,,,,SA,,,c/SA,ara,,FALSE,,,,https://www.artonline.tv/,https://i.imgur.com/DqzhSi8.png
+ARTAflam1.sa,ART Aflam 1,,,,SA,,,c/SA,ara,,FALSE,,,,https://www.artonline.tv/,https://i.imgur.com/2lKqOjj.png
+ARTAflam2.sa,ART Aflam 2,,,,SA,,,c/SA,ara,,FALSE,,,,https://www.artonline.tv/,https://i.imgur.com/st7VS0B.png
 ARTAmerica.sa,ART America,,,,SA,,,c/SA,ara,,FALSE,,,,https://www.artonline.tv/,https://i.imgur.com/QFBzdMv.png
 ArtCima.us,Art Cima,,,,US,,,c/US,ara,,FALSE,,,,https://my.dish.com/station/ARTCM,https://i.imgur.com/jY1LXxS.png
-ARTCinema.sa,ART Cinema,,,,SA,,,c/SA,ara,movies,FALSE,,,,https://www.artonline.tv/,https://i.imgur.com/71UT0Wl.png
+ARTCinema.sa,ART Cinema,,,,SA,,,c/SA,ara,movies,FALSE,,,,https://www.artonline.tv/,https://i.imgur.com/3JPkYXv.png
```
Comment (freearhey) [inline data/channels.csv:1338]: The new logo is the same image as the old one, only with extra padding.

File: data/channels.csv [hunk 3]
```diff
@@ -1331,19 +1331,19 @@ Arroyo.us,Arroyo,,,,US,US-CA,Pasadena,s/US-CA,eng,general,FALSE,,,,https://www.p
...
 ArteAtelier.it,Arte Atelier,,,,IT,,,c/IT,ita,,FALSE,,,,http://www.arteatelier.it/,https://i.imgur.com/15rMfqc.png
 ARTEFrance.fr,ARTE France,,,,FR,,,r/EUR,fra,entertainment,FALSE,,,,https://www.arte.tv/,https://i.imgur.com/d6YP7La.png
 ARTEGermany.de,ARTE Germany,ARTE Deutschland,,,DE,,,r/EUR,deu,entertainment,FALSE,,,,https://www.arte.tv/,https://i.imgur.com/d6YP7La.png
 ArteInvestimenti.it,Arte Investimenti,,,,IT,,,c/IT,ita,,FALSE,,,,https://www.arteinvestimenti.it/,https://i.imgur.com/Jy0MUb4.png
 ARTFLIXMovieClassics.us,ARTFLIX - Movie Classics,,,,US,,,c/US,eng,movies;classic,FALSE,,,,https://watch.plex.tv/live-tv/channel/artflix-movie-classics,https://i.imgur.com/5pOZQB4.png
-ARTHekayat.sa,ART Hekayat,,,,SA,,,c/SA,ara,,FALSE,,,,https://www.artonline.tv/,https://i.imgur.com/83mmS9N.png
-ARTHekayat2.sa,ART Hekayat 2,,,,SA,,,c/SA,ara,,FALSE,,,,https://www.artonline.tv/,https://i.imgur.com/fjnxvmL.png
+ARTHekayat.sa,ART Hekayat,,,,SA,,,c/SA,ara,,FALSE,,,,https://www.artonline.tv/,https://i.imgur.com/rALEQWp.png
+ARTHekayat2.sa,ART Hekayat 2,,,,SA,,,c/SA,ara,,FALSE,,,,https://www.artonline.tv/,https://imgur.com/2CrMFG9
```
Comment (freearhey) [inline data/channels.csv:1346]: The same image as the old one with extra padding.

File: data/channels.csv [hunk 4]
```diff
@@ -1331,19 +1331,19 @@ Arroyo.us,Arroyo,,,,US,US-CA,Pasadena,s/US-CA,eng,general,FALSE,,,,https://www.p
...
 Arte1.br,Arte 1,,,,BR,,,c/BR,por,,FALSE,,,,https://canalarte1.com.br/,https://i.imgur.com/Ivnk7A6.png
 ArteAtelier.it,Arte Atelier,,,,IT,,,c/IT,ita,,FALSE,,,,http://www.arteatelier.it/,https://i.imgur.com/15rMfqc.png
 ARTEFrance.fr,ARTE France,,,,FR,,,r/EUR,fra,entertainment,FALSE,,,,https://www.arte.tv/,https://i.imgur.com/d6YP7La.png
 ARTEGermany.de,ARTE Germany,ARTE Deutschland,,,DE,,,r/EUR,deu,entertainment,FALSE,,,,https://www.arte.tv/,https://i.imgur.com/d6YP7La.png
 ArteInvestimenti.it,Arte Investimenti,,,,IT,,,c/IT,ita,,FALSE,,,,https://www.arteinvestimenti.it/,https://i.imgur.com/Jy0MUb4.png
 ARTFLIXMovieClassics.us,ARTFLIX - Movie Classics,,,,US,,,c/US,eng,movies;classic,FALSE,,,,https://watch.plex.tv/live-tv/channel/artflix-movie-classics,https://i.imgur.com/5pOZQB4.png
-ARTHekayat.sa,ART Hekayat,,,,SA,,,c/SA,ara,,FALSE,,,,https://www.artonline.tv/,https://i.imgur.com/83mmS9N.png
-ARTHekayat2.sa,ART Hekayat 2,,,,SA,,,c/SA,ara,,FALSE,,,,https://www.artonline.tv/,https://i.imgur.com/fjnxvmL.png
+ARTHekayat.sa,ART Hekayat,,,,SA,,,c/SA,ara,,FALSE,,,,https://www.artonline.tv/,https://i.imgur.com/rALEQWp.png
```
Comment (freearhey) [inline data/channels.csv:1345]: Smaller than the old logo.

**Review by freearhey — CHANGES_REQUESTED** [review]
Please check the marked logos again and choose the best quality.

File: data/channels.csv (hunk 2 above)
Comment (boeder8) [reply data/channels.csv:1338]: You are right. The old logo is fine in this case.

File: data/channels.csv (hunk 3 above)
Comment (boeder8) [reply data/channels.csv:1346]: In the new logos, the text is in white which creates better contrast against darker backgrounds.

Second logo link should be: https://i.imgur.com/2CrMFG9.png

File: data/channels.csv (hunk 4 above)
Comment (boeder8) [reply data/channels.csv:1345]: I have found larger logos

ARTHekayat.sa: https://i.imgur.com/A5HZSNO.png
ARTHekayat2.sa: https://i.imgur.com/AsL7Sfl.png

File: data/channels.csv (hunk 1 above)
Comment (boeder8) [reply data/channels.csv:1335]: The text is in white which appears clearer on darker backgrounds.

Moreover, Ive found better logos from ARTonline.tv official website, should be larger and clearer than the old logos

ARTAflam1.sa
https://i.imgur.com/gPQPjAo.png

ARTAflam2.sa
https://i.imgur.com/q4Hsa4w.png

Comment (freearhey) [reply data/channels.csv:1335]: These versions are definitely better.

File: data/channels.csv (hunk 4 above)
Comment (freearhey) [reply data/channels.csv:1345]: Great job!

**Review by freearhey — APPROVED** [review]

**Pull request merged**